### PR TITLE
Update Chinese localization

### PR DESCRIPTION
**Pull request by silentdark**
Update Chinese localization

### CODE REVIEW

**Comment by sibir-ine** [comment]
@silentdark Thank you so much! If you don't mind, there are also some masterlist messages that need to be translated to Chinese. You can read more at https://github.com/loot/loot.github.io/issues/77, but, basically, we have a spreadsheet that you can fill out so you don't have to worry about making PRs to all the masterlists. You can join our [Discord](https://loot.github.io/discord/) to download it, or we can provide you with a copy.

**Comment by sibir-ine** [comment]
Also, since this is for LOOT v0.16.0, would you mind updating that in the translation file's properties?

**Comment by silentdark** [comment]
Okay, it has been changed to LOOT v0.16.0, and then I also downloaded the localization file of masterlists.

**Review by sibir-ine — APPROVED** [review]
@silentdark Thank you so much! We also have a translator role that you can apply for in #role-registration in the Discord if you'd like to be notified when we need new translations.

**Review by Ortham — CHANGES_REQUESTED** [review]
Just the one change needed, thanks.

File: resources/l10n/zh_CN/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -19,37 +19,273 @@ msgstr ""
...
+msgid "(edited)"
+msgstr "(已修改)"
+
+#: src/gui/cef/query/types/sort_plugins_query.h:54
+msgid "Sorting load order..."
+msgstr "排序加载顺序..."
+
+#: src/gui/state/loot_state.cpp:105
+msgid "Error: --auto-sort was passed but no --game parameter was provided."
+msgstr "错误: --自动排序已经完成但是没有 --提供游戏参数。"
```
[the remark this answers is not in the document]
Comment (silentdark) [reply resources/l10n/zh_CN/LC_MESSAGES/loot.po]: Got it.

File: resources/l10n/zh_CN/LC_MESSAGES/loot.po [hunk 2]
```diff
@@ -68,7 +68,7 @@ msgstr "排序加载顺序..."
 
 #: src/gui/state/loot_state.cpp:105
 msgid "Error: --auto-sort was passed but no --game parameter was provided."
-msgstr "错误: --自动排序已经完成但是没有 --提供游戏参数。"
+msgstr "Error: --auto-sort was passed but no --game parameter was provided."
```
Comment (sibir-ine) [inline resources/l10n/zh_CN/LC_MESSAGES/loot.po:71]: @silentdark Only `--auto-sort` & `--game` need to stay untranslated. The rest of the sentence can be translated.

**Review by sibir-ine — CHANGES_REQUESTED** [review]
@silentdark 4ee8ab2f6c83036a304cd8c01742b58d952d9b4e needs to be reverted (or the files restored).

**Comment by silentdark** [comment]
> @silentdark [4ee8ab2](https://github.com/loot/loot/commit/4ee8ab2f6c83036a304cd8c01742b58d952d9b4e) needs to be reverted (or the files restored).

Sorry, the new gitgub has changed so much that I am not used to it yet.

**Review by sibir-ine — APPROVED** [review]
> Sorry, the new gitgub has changed so much that I am not used to it yet.

No worries, thank you so much for your translations!

**Review by Ortham — APPROVED** [review]

**Pull request merged**